site/src/pages/components/customizableselect.mdx

```diff
@@ -177,14 +170,12 @@ This example uses the customizable `<select>` element with custom CSS to target
   <button>
     <selectedoption></selectedoption>
   </button>
-  <datalist>
-    <option>
-      ❤️ <span class=description>heart</span>
-    </option>
-    <option>
-      🙂 <span class=description>smile</span>
-    </option>
-  </datalist>
+  <option>
+    ❤️ <span class=description>heart</span>
```

Comment:
i can't imagine the intent of these options is to have them announced as "red heart heart" or "slightly smiling face smile"

if we're going to demonstrate emoji paired with text, we should probably mark the emoji as aria-hidden=true.  might be worth having a note after this to indicate the why as well (to reduce awkward redundancy, to reduce the exposed / expected accName that someone would need to figure out if using some voice control software, etc.)

Reply:
Ok, I wrapped them in `<span aria-hidden=true>`, but that also means that the aria-hidden is copied into the `<selectedoption>`. How do you think the button should be announced?

Reply:
I created a new PR to work on this example: https://github.com/openui/open-ui/pull/1079